vowpalwabbit/vw_versions.h

```diff
@@ -59,5 +59,9 @@ constexpr VW::version_struct VERSION_PASS_UINT64{8, 3, 3};
 
 /// Added serialized seen min and max labels in the --active reduction
 constexpr VW::version_struct VERSION_FILE_WITH_ACTIVE_SEEN_LABELS{9, 0, 0};
+
+/// Moved option values from command line to model data
+constexpr VW::version_struct VERSION_FILE_WITH_L1_AND_L2_STATE_IN_MODEL_DATA{8, 11, 0};
```

Comment:
Decide if okay for 9 or later and update accordingly

Reply:
We should make sure we decide this before merging the PR, since it could be a little awkward to have to change this once it is in the repo, and it could cause odd situations where a source-build does the wrong thing.

Reply:
The source build should do the right thing according to version.txt. It is just whether or not this functionality is "turned on"

Reply:
Will update this to 9.0.0 prior to merging